### ISSUE TITLE
energy metrics in home assistant?

### ISSUE THREAD

**Issue by andreasc1** [mac-zhou/midea-ac-py#109]
Hi, from the midea app we get the energy consumption of the AC. is it possible to get these metrics in home assistant's energy module?

**Comment by sklein972** [comment]
Can you please send an image where you find it? I can't see it on my app

**Comment by andreasc1** [comment]
> Can you please send an image where you find it? I can't see it on my app

you are totally right, i dont  see them. strange, i am sure i saw these statistics somewhere.. let's close this issue for now.

**Comment by savaspar** [comment]
Can I revive the issue? The metrics is when you tap on an AC and then go to the 3 dots and choose "Energy Monitor".

**Comment by andreasc1** [comment]
> Can I revive the issue? The metrics is when you tap on an AC and then go to the 3 dots and choose "Energy Monitor".

With the migration to msmarthome this does not exist anymore right?

**Comment by savaspar** [comment]
> > Can I revive the issue? The metrics is when you tap on an AC and then go to the 3 dots and choose "Energy Monitor".
> 
> With the migration to msmarthome this does not exist anymore right?

I have no idea. I use NetHome Plus, this is available for my country and the option is there on the 3 dots.
